db/migrate/20220905102928_create_features.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# creating feature table
+class CreateFeatures < ActiveRecord::Migration[5.2]
+  def change
+    create_table :features do |t|
+      t.string :name
+      t.float :unit_price
+      t.integer :max_unit_limit
```

Comment:
can be decimal

Reply:
Max unit limit I dont think can be decimal